README.md

```diff
@@ -1100,6 +1112,12 @@ The Magneticum simulations are aiming to follow the formation of cosmological st
 
 [Magneticum Publications](http://www.magneticum.org/publications.html)
 
+### Flamingo simulations
+
+Full-hydro Large-scale structure simulations with All-sky Mapping for the Interpretation of Next Generation Observations. Large cosmological simulations including baryonic matter, sub-grid physics, and neutrinos.
```

Comment:
```suggestion
The Full-hydro Large-scale structure simulations with All-Sky Mapping for the Interpretation of Next Generation Observations. Large cosmological simulations including baryonic matter, sub-grid physics, and neutrinos.
```

Reply:
Do I understand correctly that FLAMENGO is a project that uses software, but is not itself software? @nikosarcevic is this in the scope of this project?

Reply:
It's a simulation suite, comparable to Illustris, IllustrisTNG, Magneticum, Auriga  (all of which are listed there, which is why I added Flamingo)